buildscripts/dockerbuild/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM ubuntu:18.04
+RUN apt update
+RUN apt install software-properties-common -y
+RUN apt update
+RUN apt install git -y
+RUN apt install python3.8 -y
+RUN apt install python3-pip -y
+RUN apt install libpython3.8-dev -y
+RUN  ln -S -f /usr/bin/python3.8 /usr/bin/python3
+RUN apt install g++-8 -y
+RUN apt install gdb -y
+RUN update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-8 800 --slave /usr/bin/g++ g++ /usr/bin/g++-8
```

Comment:
we have the mongo toolchain, we may want to make sure thats installed http://mongodbtoolchain.build.10gen.cc/

Reply:
probably leave out the python3.8 install and symlink the mongo toolchain python instead

Reply:
mongo toolchain can be used. I was not aware of this toolchain.  Is there a way to pass a specific toolchain version (v3) for scons build?

Reply:
We don't generally distribute the toolchain we build. We use it consistently so that we have the same toolchain across all platforms and not be tied to the minimum compiler across our various platforms. But there is nothing wrong with using the system toolchain if it meets the build requirements. I'd sort of prefer we not start having others use our toolchain.